src/tribler/gui/start_gui.py

```diff
@@ -25,7 +27,7 @@
 
 
 def run_gui(api_port, api_key, root_state_dir, parsed_args):
-    logger.info('Running GUI' + ' in gui_test_mode' if parsed_args.gui_test_mode else '')
+    logger.info(f"Running GUI in {'gui_test_mode' if parsed_args.gui_test_mode else 'normal mode'}")
```

Comment:
Previously, because of the prcedence rules, an empty string was logged in normal mode. Basically:
```python
('Running GUI' + ' in gui_test_mode') if parsed_args.gui_test_mode else ('')
```